ci/scripts/common.sh

```diff
@@ -13,6 +13,8 @@ export CARGO_MAKE_PRINT_TIME_SUMMARY=true
 export MINIO_DOWNLOAD_BIN=https://ci-deps-dist.s3.amazonaws.com/minio
 export MCLI_DOWNLOAD_BIN=https://ci-deps-dist.s3.amazonaws.com/mc
 export GCLOUD_DOWNLOAD_TGZ=https://ci-deps-dist.s3.amazonaws.com/google-cloud-cli-406.0.0-linux-x86_64.tar.gz
+export CI=true
```

Comment:
https://github.com/nextest-rs/nextest/blob/6e00ddeb62229c753bf346465fbb49e093bd2f3e/nextest-runner/src/reporter.rs#L267-L274

https://github.com/zkat/is_ci/blob/bc4fda761a9810a0e967d1470772fedd78d79360/src/lib.rs#L26-L28

Reply:
Not sure if it's needed, but I think it doesn't hurt.

Reply:
weird

<img width="918" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/2670a725-99b2-4021-900f-d38a91c14fa6">